resources/js/accessible_toggle.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file
+ * Attaches behaviors for accessible modal and offcanvas triggers.
```

Comment:
if we add "collapse" in the future then we need to update this comment, so lets make it more generic